ui/src/lib/features/k8s/networks/gateways/store.ts

```diff
@@ -24,7 +24,7 @@ interface Resource extends VirtualService {
 export type Columns = ColumnWrapper<Row>
 
 export function createStore(): ResourceStoreInterface<Resource, Row> {
-  const url = `/api/v1/resources/networks/gateways?fields=.metadata.name,.metadata.namespace,.metadata.creationTimestamp,.spec.servers`
+  const url = `/api/v1/resources/networks/gateways?fields=.metadata.name,.metadata.namespace,.metadata.uid, .metadata.creationTimestamp,.spec.servers`
```

Comment:
since this list is getting a little long, could we switch to just getting all metadata?

```suggestion
  const url = `/api/v1/resources/networks/gateways?fields=.metadata,.spec.servers`
```

Reply:
> since this list is getting a little long, could we switch to just getting all metadata? we'll get a little bit of data back (labels and annotations) that we don't care about, but shouldn't cause any issues.

Updated